### ISSUE TITLE
Docker: "plugin_dir" shadowed by external volume

### ISSUE THREAD

**Issue by joschi** [Graylog2/graylog2-images#146]
The Docker image (2.0.1-2 at the time of writing) is using `/usr/share/graylog/data/plugin` as [`plugin_dir`](https://github.com/Graylog2/graylog2-images/blob/2.0.1-2/docker/config/graylog.conf#L33-L34) which means that it is shadowed the [default volume](https://github.com/Graylog2/graylog2-images/blob/2.0.1-2/docker/Dockerfile#L52).

Since the idea of Docker is having immutable images which build on each other, I suggest moving the plugin directory back to the default location (`/usr/share/graylog/plugin`) and let users create their own Docker images if they need to install additional plugins.

Example of a customized Graylog Docker image with the Beats plugin:

```
FROM Graylog2/server:2.0.2-1

RUN wget -O /usr/share/graylog/plugin/graylog-plugin-beats-1.0.3.jar https://github.com/Graylog2/graylog-plugin-beats/releases/download/1.0.3/graylog-plugin-beats-1.0.3.jar
```

**Comment by phlegx** [comment]
If one has mapped the following volume:

```
/opt/graylog/data:/usr/share/graylog/data
```

Then the `System > Pipeline` module/plugin is no more available. Talked with @joschi and he thinks this is related to this issue.

**Comment by mariussturm** [comment]
Fixed through abae3c77305da0041bcd53a44342726083b547c8
